### PR TITLE
Add support for gowin .fs files (input only)

### PR DESCRIPTION
**Pull request by danselmi**
Add support for gowin .fs files (input only) . Includes implementation, documentation and regression test.

### CODE REVIEW

File: doc/man5/srec_gowin.5 [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.\"     along with this program. If not, see
+.\"     <http://www.gnu.org/licenses/>.
+.\"
+.ds n) srec_gowin
+.TH \*(n) 5 SRecord "Reference Manual"
+.SH NAME
+srec_gowin \- Gowin bitstream file format
+.if require_index \{
+.XX "srec_gowin(5)" "Gowin bitstream file format"
+.\}
```
Comment (marcows) [inline doc/man5/srec_gowin.5:25]: - [ ] These three lines from require_index block have to be moved up between .TH and .SH when #58 has been merged.

**Review by marcows — CHANGES_REQUESTED** [review]

File: doc/man5/srec_gowin.5 (hunk 1 above)
Comment (sierrafoxtrot) [reply doc/man5/srec_gowin.5:25]: #58 is now merged so now ready for this change.

**Comment by danselmi** [comment]
I hope that I caught all spots which needed adjustments.

**Review by sierrafoxtrot — APPROVED** [review]

**Comment by sierrafoxtrot** [comment]
Looks good Daniel. Thank you!

**Pull request merged**